members/C001056.yaml

```diff
@@ -52,6 +52,11 @@ contact_form:
         values: [ "$MESSAGE" ]
         commands: ["elements[0].value = values[0].replace(/\\b(set forth in the bill)\\b/ig,'').replace(/\\b(is not)\\b/ig,'isn\\'t')"]
         required: true
+      - name: percent-symbol-replace
+        values: [ "$MESSAGE" ]
+        selectors: ["#edit-submitted-please-write-us-a-brief-message"]
+        commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/[%]/g,' percent');}"]
+        required: true
```

Comment:
Looks like you could add this onto the existing js step above

Reply:
I tried but was not replacing the char. This is what i had 
`        commands: ["elements[0].value = values[0].replace(/\\b(set forth in the bill)\\b/ig,'').replace(/\\b(is not)\\b/ig,'isn\\'t').replace(/[%]/g,' percent')"]`

Reply:
nvm. I did some more testing and it does work. I may have had an extra character or something odd in my local copy that was preventing that percent replace from working. 

I updated the yaml